kit/scrolls/src/Lists/UIMasonryList.tsx

```diff
@@ -297,16 +323,34 @@ function useVirtualization<Item>(
     React.useEffect(() => {
         /**
          * So basically when we render items for the first time,
+         * or when the data has been updated,
          * we have to calculate items that're in virtual window
          */
         const bottom = binarySearch(
-            initialWindowHeight + virtualWindowSizeRatio * initialWindowHeight,
+            initialWindowHeight +
+                virtualWindowSizeRatio * initialWindowHeight +
+                currentScrollValue.current,
```

Comment:
Not sure either it was a bug before? (when we haven't added the `currentScrollValue`)

Reply:
It was. When updating the data, the list disappeared (the viewport was calculated at the very beginning of the list)